server/routes/temporaryabsences/checkTemporaryAbsenceController.ts

```diff
@@ -16,6 +17,25 @@ export default class CheckTemporaryAbsenceController {
   public addToRoll(): RequestHandler {
     return async (req, res) => {
       const { prisonNumber } = req.params
+      const { username } = req.user
+      const { activeCaseLoadId } = res.locals.user
+      const data = await this.temporaryAbsencesService.getTemporaryAbsence(activeCaseLoadId, prisonNumber)
+
+      await this.temporaryAbsencesService.confirmTemporaryAbsence(username, prisonNumber, {
+        agencyId: activeCaseLoadId,
```

Comment:
I think it would be better to just pass in the `activeCaseLoadId` directly rather than this `AgencyId` Type. 

This leaks client implementation specific types up through the service and web layer - your controller shouldn't need to transform its data to a structure that is only required by the client if it doesn't have to.  

You could do the same in the service too - pass the agencyId straight through and build the structure solely in the client?

Reply:
done